ethicalads-theme/templates/ea/publisher-guide.html

```diff
@@ -0,0 +1,255 @@
+{% extends "page.html" %}
+
+
+{% block content %}
+<section class="py-10">
+  <div class="container">
+
+    <div class="mb-8">
+      <h1 class="font-weight-bold text-center">{{ page.title }}</h1>
+      {% if page.subtitle %}<p class="lead text-center">{{ page.subtitle }}</p>{% endif %}
+    </div>
+
+    <div class="row row-cols-1 row-cols-md-3 mb-8">
+
+      <!-- Apply -->
+      <div class="col mb-2">
+        <div class="card shadow-sm h-100">
+          <div class="card-body">
+            <h3 class="card-title">Step #1: Apply</h3>
+            <p class="card-text">The first step is to <a href="/publishers/#inbound-form">apply to become a publisher</a>. We are actively seeking developer-focused sites and projects with 50k+ monthly pageviews. Once accepted, you'll have access to our publisher platform.</p>
+
+            <hr>
+
+            <div class="text-center py-3">
+              <img src="/theme/img/illustrations/illustration-3.png" alt="Apply to EthicalAds" class="w-75">
+            </div>
+
+          </div>
+        </div>
+      </div>
+
+      <!-- Setup -->
+      <div class="col mb-2">
+        <div class="card shadow-sm h-100">
+          <div class="card-body">
+            <h3 class="card-title">Step #2: Setup</h3>
+            <p class="card-text">Setup the EthicalAds <a href="https://ethical-ad-client.readthedocs.io/">ad client</a> on your site with your custom ID from our publisher platform. Out of the box you'll serve unpaid ads until your site is approved for paid ads.</p>
```

Comment:
```suggestion
            <p class="card-text">Setup the EthicalAds <a href="https://ethical-ad-client.readthedocs.io/">ad client</a> on your site with your custom ID from our publisher dashboard. Out of the box you'll serve unpaid ads until your site is approved for paid ads.</p>
```

I feel like platform is a weird word, and dashboard is more clear, but not 100% sure that's the best term either.